Podfile

```diff
@@ -91,7 +91,7 @@ def shared_style_pods
 end
 
 def gutenberg_pods
-  gutenberg tag: 'v1.88.1-alpha1'
+  gutenberg commit: '581dfd445660821663474c13f9b55c3057aaf182'
```

Comment:
Friendly reminder: we'll need to update this to the Gutenberg release tag once it is published.